### PR TITLE
add generate makefile command

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

**Comment by dmajda** [comment]
LGTM beside the line :-)

@mvidner Could you check whether the generated stuff makes sense from your POV?

File: yk [hunk 1]
```diff
@@ -161,8 +164,83 @@ class YastModule
...
+      :key      => "schemafiles_DATA",
+      :key_line => "schemafilesdir = $(schemadir)/autoyast/rnc\nschemafiles_DATA"
+    },
+    {
+      :glob     => "bin/*",
+      :key      => "ybin_SCRIPTS"
+    },
+    {
+      :glob     => "data/*",
+      :key      => "ydata_DATA"
```
Comment (mvidner) [inline yk:236]: yast2-country uses a subdirectory, data/languages, yast2-testsuite uses a subdir too, but I think it is OK to leave these out as long as I can special case it somehow

File: yk [hunk 2]
```diff
@@ -161,8 +164,83 @@ class YastModule
...
+
   private
 
+  def makefile_entry(key, values)
+    "\n#{key} = \\\n  " + values.join(" \\\n  ") + "\n"
+  end
+
+  MAKEFILE_DESCRIPTION = [
+    {
+      :glob     => 'modules/**/*\.{ycp,ybc,rb,py,pl,sh}',
```
Comment (mvidner) [inline yk:206]: This is a glob, not a regex, so why are we escaping dots?

Comment (mvidner) [inline yk:206]: About the double star: will automake recreate the subdir hierarchy (esp. `modules/YaPI/...`) or will it flatten it?

File: yk (hunk 1 above)
Comment (jreidinger) [inline yk:236]: trick is that it is quite problematic to express it properly in autotools. I expect there some problem, so we will see how it works. I check country you mention and it contain special handling for it. It is not problem to edit generated Makefile to fit needs. It is there just to help with common module

File: yk (hunk 2 above)
Comment (jreidinger) [inline yk:206]: that is good question and we should try it. escaped dot is just mistake.

**Comment by mvidner** [comment]
Looks good, but I don't have time to try it out, I have to 今すぐ行きます、ごめんなさい

**Pull request merged**